WMFData/Sources/WMFData/Data Controllers/Year In Review/WMFYearInReviewDataController.swift

```diff
@@ -454,12 +542,30 @@ public class WMFYearInReviewDataController {
         }
     }
     
-    public func fetchUserContributionsCount(username: String, project: WMFProject, completion: @escaping (Result<(Int, Bool), Error>) -> Void) {
+    public func fetchUserContributionsCount(username: String, project: WMFProject?) async throws -> (Int, Bool) {
+        return try await withCheckedThrowingContinuation { continuation in
+            fetchUserContributionsCount(username: username, project: project) { result in
+                switch result {
+                case .success(let successResult):
+                    continuation.resume(returning: successResult)
+                case .failure(let error):
+                    continuation.resume(throwing: error)
+                }
+            }
+        }
+    }
+    
+    public func fetchUserContributionsCount(username: String, project: WMFProject?, completion: @escaping (Result<(Int, Bool), Error>) -> Void) {
         guard let service = service else {
             completion(.failure(WMFDataControllerError.mediaWikiServiceUnavailable))
             return
         }
         
+        guard let project = project else {
+            completion(.failure(WMFDataControllerError.mediaWikiServiceUnavailable))
+            return
+        }
+        
         let ucStartDate = "2024-11-01T22:20:13.000Z"
         let ucEndDate = "2024-01-01T22:20:13.000Z"
```

Comment:
Then use your config date string parameters here instead of hardcoding them. Note that the API accepts them in an opposite way:

```
let ucStartDate = dataPopulationEndDateString
let ucEndDate = dataPopulationStartDateString
```